br/pkg/lightning/restore/restore.go

```diff
@@ -975,6 +975,11 @@ func (rc *Controller) saveStatusCheckpoint(ctx context.Context, tableName string
 	case !common.IsContextCanceledError(err):
 		merger.SetInvalid()
 		rc.errorSummaries.record(tableName, err, statusIfSucceed)
+	case utils.IsRetryableError(err), utils.MessageIsRetryableStorageError(err.Error()):
```

Comment:
this case should move before `case !common.IsContextCanceledError(err):`, else it's never run